requirements.txt

```diff
@@ -3,4 +3,5 @@ flake8
 python-novaclient==7.0.0
 PyYAML
 texttable
-paramiko==2.1.2
\ No newline at end of file
+# temporarily disabled
```

Comment:
We will separate openstack tests from salt-based checks in future, so we don't need benchmark/openstack_requirements.txt.